Demo/Application/Base/Settings/BraintreeDemoSettings.swift

```diff
@@ -11,7 +11,6 @@ enum BraintreeDemoEnvironment: Int {
 enum BraintreeDemoAuthType: Int {
     case clientToken
     case tokenizationKey
-    case payPalIDToken
```

Comment:
We will also need to remove this from the [settings menu here](https://github.com/braintree/braintree_ios/blob/7ccda0f255230b8d427258766cb8041ba5ead5a4/Demo/Application/Base/Settings/Settings.bundle/Root.plist#L106)